helmfile/values/podsecuritypolicies/common/rook-ceph.yaml.gotmpl

```diff
@@ -0,0 +1,234 @@
+{{- if .Values.rookCeph.enabled }}
+constraints:
+  rook-ceph:
+    rook-ceph-operator:
+      podSelectorLabels:
+        app: rook-ceph-operator
+      allow:
+        privileged: true
+        allowPrivilegeEscalation: true
+        volumes:
+          - emptyDir
+          - projected
+    csi-rbdplugin-provisioner:
+      podSelectorLabels:
+        app: csi-rbdplugin-provisioner
+      allow:
+        privileged: true
+        allowPrivilegeEscalation: true
+        volumes:
+          - emptyDir
+          - projected
+          - hostPath
+        allowedHostPaths:
+          - pathPrefix: "/dev"
+          - pathPrefix: "/sys"
+          - pathPrefix: "/lib/modules"
+        runAsUser:
+          rule: RunAsAny
+        runAsGroup:
+          rule: RunAsAny
+        supplementalGroups:
+          rule: RunAsAny
+        fsGroup:
+          rule: RunAsAny
+        allowedCapabilities:
+          - SYS_ADMIN
```

Comment:
Do we need to touch anything with capabilities as we are running a privileged container?
Privileged should lift any restrictions there are on capabilities.

Reply:
This was mostly to avoid violating the constraint `capabilities-restricted` which requires containers to drop all capabilities if none are specified under allowedCapabilities.

Reply:
Should be able to change this once [these](https://github.com/elastisys/compliantkubernetes-apps/pull/1559/files) changes are merged